js/okx.js

```diff
@@ -3173,6 +3176,11 @@ module.exports = class okx extends Exchange {
             // 'before': 'id', // return records newer than the requested bill id
             // 'limit': 100, // default 100, max 100
         };
+        const [ marginMode, marginModeQuery ] = this.handleMarginModeAndParams ('fetchLedger', params);
+        if (marginMode !== undefined) {
```

Comment:
@Dan-krm can we replicate the margin-logic applied in fetchLeverage here, for the sake of consistency?

Reply:
Okay I replicated the logic except I didn't default to cross margin since the mgnMode parameter is optional here.

Reply:
@Dan-krm you did well, tkx!